33
subject 172332
loaded saved variables
version 3
134
/users/k1201869/miniconda3/envs/python38/lib/python3.8/site-packages/distributed/node.py:151: UserWarning: Port 8787 is already in use.
Perhaps you already have a cluster running?
Hosting the HTTP server on port 39344 instead
  warnings.warn(
parrallel submitted 10:20:03
parrallel finished 10:24:43
estimation_imag_fix_testretest.py:217: ComplexWarning: Casting complex values to real discards the imaginary part
  inv_JinvCeJ[i,:] = J[:,i].T * inv_DiagCe
estimation_imag_fix_testretest.py:230: ComplexWarning: Casting complex values to real discards the imaginary part
  P[:,i] = (J[:,i]*np.squeeze(inv_DiagCe))
estimation_imag_fix_testretest.py:250: ComplexWarning: Casting complex values to real discards the imaginary part
  g[i] = -0.5*np.exp(lembda[i])*np.trace(P[T*i:T*i+T][:, T*i:T*i+T])
estimation_imag_fix_testretest.py:255: ComplexWarning: Casting complex values to real discards the imaginary part
  g[i] = g[i] + g_rest
/users/k1201869/wang_model/code/model_fitting/wang_functions_imag_fix.py:502: ComplexWarning: Casting complex values to real discards the imaginary part
  tr[i] = np.sum(temp)
/users/k1201869/miniconda3/envs/python38/lib/python3.8/site-packages/numpy/core/fromnumeric.py:87: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray
  return ufunc.reduce(obj, axis, dtype, out, **passkwargs)
[[-2.09804307]]
estimation_imag_fix_testretest.py:309: ComplexWarning: Casting complex values to real discards the imaginary part
  JinvCe[i,:] = J[:,i] * inv_DiagCe #;% J'%invCe <----- p x nT
estimation_imag_fix_testretest.py:320: ComplexWarning: Casting complex values to real discards the imaginary part
  A_FDK[:,ppi] = np.squeeze(A) + np.squeeze(d_A) # %newton-gauss, fisher scoring, update Para_E
/users/k1201869/wang_model/code/model_fitting/wang_functions_imag_fix.py:517: FutureWarning: `rcond` parameter will change to the default of machine precision times ``max(M, N)`` where M and N are the input matrix dimensions.
To use the future default and silence this warning we advise to pass `rcond=None`, to keep using the old, explicitly pass `rcond=-1`.
  sol = np.linalg.lstsq(A, b)[0]    # not under-determined
estimation_imag_fix_testretest.py:327: ComplexWarning: Casting complex values to real discards the imaginary part
  A_FDK[:,ppi] = np.squeeze(A) + d_A #newton-gauss, fisher scoring, update Para_E
distributed.client - ERROR - Failed to reconnect to scheduler after 10.00 seconds, closing client
_GatheringFuture exception was never retrieved
future: <_GatheringFuture finished exception=CancelledError()>
asyncio.exceptions.CancelledError
estimation_imag_fix_testretest.py:331: ComplexWarning: Casting complex values to real discards the imaginary part
  h_output_FDK[:,ppi], CC_check_FDK[ppi] = funcP(np.atleast_2d(Para_E_new).T)
estimation_imag_fix_testretest.py:334: ComplexWarning: Casting complex values to real discards the imaginary part
  dlddpara_FDK[:,:,ppi] = dlddpara
estimation_imag_fix_testretest.py:335: ComplexWarning: Casting complex values to real discards the imaginary part
  dldpara_FDK[:,ppi] = dldpara
[[-2.03602504]]
imaginary won
goodness of fit correlation [0.09735354]
0.37406733846144297
[5.09347295e-01 5.16810593e-01 4.94937038e-01 5.69710580e-01
 4.97472560e-01 5.44757844e-01 5.75467033e-01 5.45441831e-01
 5.22725837e-01 5.73791907e-01 5.17724833e-01 5.42037933e-01
 5.28007669e-01 5.29337080e-01 5.12159659e-01 5.15383701e-01
 5.05094196e-01 5.09299465e-01 5.04005669e-01 5.01359273e-01
 5.26630466e-01 5.42928854e-01 4.94986188e-01 6.01224911e-01
 5.20342860e-01 5.13937856e-01 5.05788338e-01 5.69735268e-01
 5.55758948e-01 5.32273029e-01 5.03918029e-01 5.04532771e-01
 5.06783287e-01 5.11636218e-01 5.05123422e-01 5.26916494e-01
 5.12922240e-01 5.03597482e-01 4.97558097e-01 5.40167980e-01
 5.18378181e-01 5.40768542e-01 5.15282881e-01 5.71229728e-01
 5.20375157e-01 5.63210997e-01 5.22236140e-01 5.54082814e-01
 5.18688938e-01 5.04418120e-01 5.19341751e-01 5.09929161e-01
 5.31255697e-01 5.55812882e-01 5.08734886e-01 5.74679882e-01
 5.15992036e-01 6.30863513e-01 5.11478854e-01 5.08300064e-01
 4.97560629e-01 6.36578781e-01 5.24364583e-01 5.35629867e-01
 5.09435739e-01 5.10743183e-01 5.08818615e-01 5.74706991e-01
 3.37014770e-01 3.43164606e-01 3.32452772e-01 3.68769788e-01
 3.16180507e-01 3.38604201e-01 3.31849225e-01 3.43518041e-01
 3.44950369e-01 3.45321897e-01 3.41928666e-01 3.57728756e-01
 3.34360740e-01 3.40443589e-01 3.46858119e-01 3.57047559e-01
 3.38750113e-01 3.29857756e-01 3.32121880e-01 3.46818255e-01
 3.41362787e-01 3.49028628e-01 3.34515490e-01 3.30657243e-01
 3.50595711e-01 3.38709687e-01 3.15277392e-01 3.24556607e-01
 3.50274613e-01 3.39176545e-01 3.25857151e-01 3.22743993e-01
 3.40726075e-01 3.38956902e-01 3.48588721e-01 3.32457373e-01
 3.35257519e-01 3.62121361e-01 3.11421416e-01 3.40890176e-01
 3.34265506e-01 3.39763043e-01 3.65892059e-01 3.51136698e-01
 3.31962136e-01 3.70264932e-01 3.34931814e-01 3.42929203e-01
 3.45648722e-01 3.55962601e-01 3.36876864e-01 3.44176055e-01
 3.37066186e-01 3.55514202e-01 3.45548373e-01 3.49077466e-01
 3.43648000e-01 3.29967496e-01 3.39267293e-01 3.44451966e-01
 3.15445197e-01 3.23153859e-01 3.55104010e-01 3.44227251e-01
 3.35777720e-01 3.22845037e-01 3.54928277e-01 3.39749835e-01
 1.55735588e+00 9.98604459e-04]
version 4
0
/users/k1201869/miniconda3/envs/python38/lib/python3.8/site-packages/distributed/node.py:151: UserWarning: Port 8787 is already in use.
Perhaps you already have a cluster running?
Hosting the HTTP server on port 40223 instead
  warnings.warn(
parrallel submitted 10:25:32
parrallel finished 10:30:07
estimation_imag_fix_testretest.py:217: ComplexWarning: Casting complex values to real discards the imaginary part
  inv_JinvCeJ[i,:] = J[:,i].T * inv_DiagCe
estimation_imag_fix_testretest.py:230: ComplexWarning: Casting complex values to real discards the imaginary part
  P[:,i] = (J[:,i]*np.squeeze(inv_DiagCe))
estimation_imag_fix_testretest.py:250: ComplexWarning: Casting complex values to real discards the imaginary part
  g[i] = -0.5*np.exp(lembda[i])*np.trace(P[T*i:T*i+T][:, T*i:T*i+T])
estimation_imag_fix_testretest.py:255: ComplexWarning: Casting complex values to real discards the imaginary part
  g[i] = g[i] + g_rest
/users/k1201869/wang_model/code/model_fitting/wang_functions_imag_fix.py:502: ComplexWarning: Casting complex values to real discards the imaginary part
  tr[i] = np.sum(temp)
/users/k1201869/miniconda3/envs/python38/lib/python3.8/site-packages/numpy/core/fromnumeric.py:87: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray
  return ufunc.reduce(obj, axis, dtype, out, **passkwargs)
[[-2.07972994]]
estimation_imag_fix_testretest.py:309: ComplexWarning: Casting complex values to real discards the imaginary part
  JinvCe[i,:] = J[:,i] * inv_DiagCe #;% J'%invCe <----- p x nT
estimation_imag_fix_testretest.py:320: ComplexWarning: Casting complex values to real discards the imaginary part
  A_FDK[:,ppi] = np.squeeze(A) + np.squeeze(d_A) # %newton-gauss, fisher scoring, update Para_E
/users/k1201869/wang_model/code/model_fitting/wang_functions_imag_fix.py:517: FutureWarning: `rcond` parameter will change to the default of machine precision times ``max(M, N)`` where M and N are the input matrix dimensions.
To use the future default and silence this warning we advise to pass `rcond=None`, to keep using the old, explicitly pass `rcond=-1`.
  sol = np.linalg.lstsq(A, b)[0]    # not under-determined
estimation_imag_fix_testretest.py:327: ComplexWarning: Casting complex values to real discards the imaginary part
  A_FDK[:,ppi] = np.squeeze(A) + d_A #newton-gauss, fisher scoring, update Para_E
distributed.client - ERROR - Failed to reconnect to scheduler after 10.00 seconds, closing client
_GatheringFuture exception was never retrieved
future: <_GatheringFuture finished exception=CancelledError()>
asyncio.exceptions.CancelledError
estimation_imag_fix_testretest.py:331: ComplexWarning: Casting complex values to real discards the imaginary part
  h_output_FDK[:,ppi], CC_check_FDK[ppi] = funcP(np.atleast_2d(Para_E_new).T)
estimation_imag_fix_testretest.py:334: ComplexWarning: Casting complex values to real discards the imaginary part
  dlddpara_FDK[:,:,ppi] = dlddpara
estimation_imag_fix_testretest.py:335: ComplexWarning: Casting complex values to real discards the imaginary part
  dldpara_FDK[:,ppi] = dldpara
[[-2.0177259]]
imaginary won
goodness of fit correlation [0.03714768]
1
/users/k1201869/miniconda3/envs/python38/lib/python3.8/site-packages/distributed/node.py:151: UserWarning: Port 8787 is already in use.
Perhaps you already have a cluster running?
Hosting the HTTP server on port 42222 instead
  warnings.warn(
parrallel submitted 10:30:55
parrallel finished 10:34:42
estimation_imag_fix_testretest.py:217: ComplexWarning: Casting complex values to real discards the imaginary part
  inv_JinvCeJ[i,:] = J[:,i].T * inv_DiagCe
estimation_imag_fix_testretest.py:230: ComplexWarning: Casting complex values to real discards the imaginary part
  P[:,i] = (J[:,i]*np.squeeze(inv_DiagCe))
estimation_imag_fix_testretest.py:250: ComplexWarning: Casting complex values to real discards the imaginary part
  g[i] = -0.5*np.exp(lembda[i])*np.trace(P[T*i:T*i+T][:, T*i:T*i+T])
estimation_imag_fix_testretest.py:255: ComplexWarning: Casting complex values to real discards the imaginary part
  g[i] = g[i] + g_rest
/users/k1201869/wang_model/code/model_fitting/wang_functions_imag_fix.py:502: ComplexWarning: Casting complex values to real discards the imaginary part
  tr[i] = np.sum(temp)
/users/k1201869/miniconda3/envs/python38/lib/python3.8/site-packages/numpy/core/fromnumeric.py:87: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray
  return ufunc.reduce(obj, axis, dtype, out, **passkwargs)
[[-1.85568096]]
estimation_imag_fix_testretest.py:309: ComplexWarning: Casting complex values to real discards the imaginary part
  JinvCe[i,:] = J[:,i] * inv_DiagCe #;% J'%invCe <----- p x nT
/users/k1201869/wang_model/code/model_fitting/wang_functions_imag_fix.py:517: FutureWarning: `rcond` parameter will change to the default of machine precision times ``max(M, N)`` where M and N are the input matrix dimensions.
To use the future default and silence this warning we advise to pass `rcond=None`, to keep using the old, explicitly pass `rcond=-1`.
  sol = np.linalg.lstsq(A, b)[0]    # not under-determined
estimation_imag_fix_testretest.py:320: ComplexWarning: Casting complex values to real discards the imaginary part
  A_FDK[:,ppi] = np.squeeze(A) + np.squeeze(d_A) # %newton-gauss, fisher scoring, update Para_E
estimation_imag_fix_testretest.py:327: ComplexWarning: Casting complex values to real discards the imaginary part
  A_FDK[:,ppi] = np.squeeze(A) + d_A #newton-gauss, fisher scoring, update Para_E
distributed.client - ERROR - Failed to reconnect to scheduler after 10.00 seconds, closing client
_GatheringFuture exception was never retrieved
future: <_GatheringFuture finished exception=CancelledError()>
asyncio.exceptions.CancelledError
estimation_imag_fix_testretest.py:331: ComplexWarning: Casting complex values to real discards the imaginary part
  h_output_FDK[:,ppi], CC_check_FDK[ppi] = funcP(np.atleast_2d(Para_E_new).T)
estimation_imag_fix_testretest.py:334: ComplexWarning: Casting complex values to real discards the imaginary part
  dlddpara_FDK[:,:,ppi] = dlddpara
estimation_imag_fix_testretest.py:335: ComplexWarning: Casting complex values to real discards the imaginary part
  dldpara_FDK[:,ppi] = dldpara
[[-1.91817423]]
real won
goodness of fit correlation [0.06736493]
2
/users/k1201869/miniconda3/envs/python38/lib/python3.8/site-packages/distributed/node.py:151: UserWarning: Port 8787 is already in use.
Perhaps you already have a cluster running?
Hosting the HTTP server on port 39541 instead
  warnings.warn(
parrallel submitted 10:35:30
